.github/workflows/deploy-eks.yaml

```diff
@@ -85,3 +86,11 @@ jobs:
       - name: Get operator logs
         run: kubectl logs --tail 100 -ntesting -loperator.juju.is/name=mlflow-server
         if: failure()
+
+  delete-unattached-volumes:
+    if: always()
+    uses: ./.github/workflows/delete-aws-volumes.yaml
+    secrets: inherit
+    with:
+      region: eu-central-1
+    needs: [deploy-ckf-to-eks]
```

Comment:
is `needs` necessary? won't this always run after the `deploy-ckf-to-eks` action?